modules/layers/src/text-layer/multi-icon-layer/multi-icon-layer-fragment.glsl.ts

```diff
@@ -28,7 +28,7 @@ uniform sampler2D iconsTexture;
 uniform float gamma;
 uniform bool sdf;
 uniform float alphaCutoff;
-uniform float buffer;
+uniform float sdfBuffer;
```

Comment:
headless-gl does not compile if a uniform is named `buffer`. Seems to trigger some internal reserved word.